t/ui/25-developer_mode.t

```diff
@@ -318,6 +318,7 @@ subtest 'expand developer panel' => sub {
     subtest 'behavior when changes have not been confirmed' => sub {
         my @options = $driver->find_elements('#developer-pause-at-module option');
 
+        ok $options[4], 'item visible' or return undef;
```

Comment:
Isn't "item visible" misleading? At least I see no check to confirm that the element is visible.

Reply:
Indeed. This is just to confirm that i.e. the 4th option is there to avoid trying to click an `undef` value.